templates/logging-sidecar-configmap.yaml

```diff
@@ -24,44 +24,44 @@ data:
           - "${SIDECAR_LOGS}/*.log"
         read_from: beginning
     transforms:
-      transform_syslog:
+      transform_airflow_logs:
         type: remap
         inputs:
           - airflow_log_files
         source: |
           .component = "${COMPONENT:--}"
           .workspace = "${WORKSPACE:--}"
           .release = "${RELEASE:--}"
-          .date_nano = parse_timestamp!(.@timestamp, format: "%Y-%m-%dT%H:%M:%S.%f%Z")
+          .date_nano = parse_timestamp!(.@timestamp, format: "%Y-%m-%dT%H:%M:%S.%f%Z") ?? now()
```

Comment:
I'm not sure this is needed. Maybe we can put this one back to the way it was before. What do you think @pgvishnuram ?